### ISSUE TITLE
Werkstatt: make species, culture, profession and names match to each other

### ISSUE THREAD

**Issue by roland-gsell** [Plushtoast/dsa5-foundryVTT#965]
It would be very nice, if the randomizer could choose a species and then matching cultures, professions and reasonable names.

Getting elves as species, a dwarf culture, sugar baker as profession and an utulu name might be a challenge as player character, but is less useful for immediate use as an ad-hoc encounter.
Of course all of these options can be changed individually, but it would be so much more convenient to get credible characters directly.
Maybe the complete randomness can still be useful and kept as an option.

**Comment by Plushtoast** [comment]
we already considered this, but most of the professions and cultures don't have enough data on allowed combinations.
So making this will be somehow vague...

**Comment by roland-gsell** [comment]
Ok, I see why this is impossible to implement exactly according to the rule book.
Even the vague way might be difficult:
Any given option would need to take into account how likely it would be to match with the already chosen characteristics.
One should write a demographic analysis about aventuria. :-)
If you could image how I could help to make this happen, just tell me what to do. Maybe probability tables or something like that. If it's worth the time investment is another story.

**Comment by Plushtoast** [comment]
Yes. There is no appropriate data available. So this data would have to be created, which is probably out of scope in the near future.
